gateway/ln-gateway/Cargo.toml

```diff
@@ -22,6 +22,7 @@ path = "src/bin/cln_extension.rs"
 
 [dependencies]
 anyhow = "1.0.66"
+async-stream = "0.3.3"
```

Comment:
not a fan of this additional dep, will it go away once the TODO further down is addressed?

https://github.com/fedimint/fedimint/pull/1295/commits/3d67279b989159f09e5add6812077e3b1f873ec0#diff-11314b558aa2437b3f9376371d29c660caf580b98f1e52fe9f7f548d7cc851d0R206-R207

Reply:
Dependency on `async_stream` can go away now if I find an easier way of producing Streams.
Addressing TODO opens up optimizations on processing, where we can send the same cln rpc instance to different threads to execute pay invoice requests